src/elements/file-selector/readme.md

```diff
@@ -84,6 +84,9 @@ It's suggested to have a different value for each variant, e.g.:
 ></sbb-file-selector>
 ```
 
+<!-- Override
+  @type value => File[] \| null
```

Comment:
needs logic to have effect (only reminder)

Reply:
I'll add a reminder in the description 👍

Reply:
@kyubisation brought the idea of creating a class decorator where we could define a kind of map with overrides instead of doing it in the readme files. What do you think?

Reply:
We go for the decorator approach (probably in a following pr)